src/db/models/tokens.ts

```diff
@@ -9,7 +9,7 @@ export interface tokensAttributes {
   id: number;
   name: string;
   symbol: string;
-  address: string;
+  address: string | Sequelize.WhereOptions<tokensAttributes>;
```

Comment:
You can't do this here, because model files could be overwritten by the `update-models` script. I think it's better if you use `WhereOptions` on the query. As we are using here:
https://github.com/layerx-labs/webnetwork/blob/master/pages/api/search/issues/index.ts#LL24C73-L24C73